config.lua

```diff
@@ -24,6 +24,7 @@ maxPlayers = 0
 motd = "Welcome to The Forgotten Server!"
 onePlayerOnlinePerAccount = true
 allowClones = false
+allowWalkthrough = true
```

Comment:
Shouldn't this be false by default? Sefaults should resemble vanilla in my opinion